Tests/SonarQube.TeamBuild.PreProcessor.Tests/ArgumentProcessorTests.cs

```diff
@@ -58,6 +58,37 @@ public void PreArgProc_MissingArguments()
         }
 
         [TestMethod]
+        public void PreArgProc_UrlIsRequired()
```

Comment:
The ArgumentProcessor is responsible for validating the URL; new test to cover this.
The bootstrapper and pre-processor both contain similar logic to check the URL is supplied.